src/components/common/Navbar.jsx

```diff
@@ -20,6 +20,20 @@ export class NavBar extends Component {
     modalIsOpen: false
   };
 
+  setUsername = (userData) => {
+    const { name, username } = userData;
+    let displayName;
+    if (name) {
+      const display = name.split(' ')[0];
+      displayName = display;
+    } else if (username) {
+      displayName = username.replace(/\d+/, '');
```

Comment:
I believe the logic here is to remove the first set of numbers it finds in the string. Is it not possible for the username to be strictly numbers?